bus-board/src/components/BusDataRow.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import {IJsonBus} from "../modules/IJsonBus";
+
+export const BusDataRow: React.FC<{ busData: IJsonBus }> = ({busData}) => {
+    return (

```

Comment:
We can further destructure this object in the component:
`const { lineName, destinationName, towards, timeToStation } = busData;`